src/static/js/routes/the-bureau/bureau-structure/index.js

```diff
@@ -8,22 +8,29 @@
 var $ = require( 'jquery' );
 require( 'slick' );
 
-var ContentSlider = require( '../../../modules/ContentSlider' );
-var BreakpointHandler = require( '../../../modules/BreakpointHandler' );
+var ContentSlider = require( '../../../modules/classes/ContentSlider' );
+var BreakpointHandler = require( '../../../modules/classes/BreakpointHandler' );
 
 function init() {
-  if ( !$( 'html' ).hasClass( 'lt-ie9' ) ) {
-    var settings = {
-      breakpoint: 599,
-      type:       'max',
-      enter:      _createSlider,
-      leave:      _destroySlider
-    };
-
-    var handler = new BreakpointHandler( settings );
+  if ( $( 'html' ).hasClass( 'lt-ie9' ) ) {
+    return;
   }
 
+  // TODO: Remove this when per-page JS is introduced.
+  if ( document.querySelectorAll( '#content-slider' ).length === 0 ) {
+    return;
+  }
+
+  var bpBettings = {
```

Comment:
Typo fixed in https://github.com/cfpb/cfgov-refresh/commit/c3cd0a7555ed476357d10acb073619fa0ab60cb6 

As to the original comment. I was confused between which page this was referencing. `/the-bureau/` init code has been updated, while the `/the-bureau/bureau-structure/` uses the ContentSlider, not the MobileCarousel, so an update there is out of scope.